### PR TITLE
Add styling for template values [SCI-5630]

### PR DESCRIPTION
**Pull request by aignatov-bio**
Jira ticket: [SCI-5630](https://biosistemika.atlassian.net/browse/SCI-5630)

### What was done
Add styling for template values

### CODE REVIEW

File: app/components/reports/team_member_input_component.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <% if @editing %>
```
Comment (okriuchykhin) [inline app/components/reports/team_member_input_component.html.erb:1]: It was moved to ProjectMembersInputComponent

File: app/assets/javascripts/reports/new.js [hunk 2]
```diff
@@ -1184,6 +1184,15 @@ function reportHandsonTableConverter() {
         let template = $('#templateSelector').val();
         $.get($('#templateSelector').data('valuesEditorPath'), { template: template }, function(result) {
           $('.report-template-values-container').html(result.html);
+
+          $.each($('.report-template-values-container .values-container select'), function(i, select) {
```
Comment (okriuchykhin) [inline app/assets/javascripts/reports/new.js:1188]: I think it is already initialized

Comment (aignatov-bio) [reply app/assets/javascripts/reports/new.js:1188]: It is for dropdowns inside template values (user selection)

Comment (aignatov-bio) [reply app/assets/javascripts/reports/new.js:1188]: Ah i see. I will rebase.

**Review by okriuchykhin — APPROVED** [review]

**Pull request merged**